zone/mob.cpp

```diff
@@ -6608,3 +6608,7 @@ void Mob::SetBucket(std::string bucket_name, std::string bucket_value, std::stri
 	std::string full_bucket_name = fmt::format("{}-{}", GetBucketKey(), bucket_name);
 	DataBucket::SetData(full_bucket_name, bucket_value, expiration);
 }
+
+bool Mob::IsValidXTarget() const {
+	return (!GetID() == 0 || !IsCorpse());
```

Comment:
The bool check on `GetID` is a bit confusing. Why don't we just `GetID()` > 0 in this case?

Reply:
Honestly, this was a case of trying to accurately reflect what was in the client.cpp code that previously determined validity of XTargets.  I will gladly make this change.